app/knights/class_Knights.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import annotations
+
+
+class KnightInstances:
```

Comment:
```suggestion
class Knight:
```
Why you add suffux Instances?

Reply:
I understand that this is not a good name, I fixed it.